### PR TITLE
optional SA creation

### PR DESCRIPTION
**Pull request by prudnitskiy**
## Description of the change

This MR allows changing of serviceAccount creation for policy-controller:
- allow disabling serviceAccount creation
- allows changing serviceAccount name

## Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). Where applicable, update and bump the versions in any associated umbrella chart
- [X] Variables are documented in the `values.yaml` and added to the README.md. The [helm-docs](https://github.com/norwoodj/helm-docs) utility can be used to generate the necessary content. Use `helm-docs --dry-run` to preview the content.
- [?] JSON Schema generated.
- [?] List tests pass for Chart using the [Chart Testing](https://github.com/helm/chart-testing) tool and the `ct lint` command.

### CODE REVIEW

File: charts/policy-controller/README.md [hunk 1]
```diff
@@ -58,7 +60,7 @@ The Helm chart for Policy  Controller
 | webhook.extraArgs | object | `{}` |  |
 | webhook.image.pullPolicy | string | `"IfNotPresent"` |  |
 | webhook.image.repository | string | `"ghcr.io/sigstore/policy-controller/policy-controller"` |  |
-| webhook.image.version | string | `"sha256:947693aa3a536992bc89f3c7ded8a7707b26cd4518972f293edd3e57e112438e"` | `"v0.7.0"` |
```
Comment (hectorj2f) [inline charts/policy-controller/README.md:61]: please, keep the `v0.7.0` description

File: charts/policy-controller/README.md [hunk 2]
```diff
@@ -27,12 +27,12 @@ The Helm chart for Policy  Controller
 | cosign.webhookName | string | `"policy.sigstore.dev"` |  |
 | imagePullSecrets | list | `[]` |  |
 | installCRDs | bool | `true` |  |
-| policywebhook.configData | object | `{}` | Set the data of the `policy-config-controller` configmap |
```
Comment (hectorj2f) [inline charts/policy-controller/README.md:30]: please, keep this description too.

**Review by hectorj2f — CHANGES_REQUESTED** [review]

File: charts/policy-controller/README.md (hunk 2 above)
Comment (prudnitskiy) [reply charts/policy-controller/README.md]: Updated.

File: charts/policy-controller/README.md (hunk 1 above)
Comment (prudnitskiy) [reply charts/policy-controller/README.md]: Updated

File: charts/policy-controller/README.md [hunk 3]
```diff
@@ -32,7 +32,7 @@ The Helm chart for Policy  Controller
 | policywebhook.extraArgs | object | `{}` |  |
 | policywebhook.image.pullPolicy | string | `"IfNotPresent"` |  |
 | policywebhook.image.repository | string | `"ghcr.io/sigstore/policy-controller/policy-webhook"` |  |
-| policywebhook.image.version | string | `"sha256:3a3581032ff69991ddd9f19faf8acd059c8fb0ba8d3b0164a13b99a096880c84"` | `"v0.7.0"` |
+| policywebhook.image.version | string | `"sha256:3a3581032ff69991ddd9f19faf8acd059c8fb0ba8d3b0164a13b99a096880c84"` |  |
```
Comment (hectorj2f) [inline charts/policy-controller/README.md:35]: v0.7.0 is still missing here :).

Comment (hectorj2f) [reply charts/policy-controller/README.md:35]: the rest lgtm.

**Review by hectorj2f — DISMISSED** [review]

**Comment by hectorj2f** [comment]
One last thing, please sign off your commits :).

File: charts/policy-controller/README.md (hunk 3 above)
Comment (prudnitskiy) [reply charts/policy-controller/README.md]: fixed, thank you

**Review by hectorj2f — APPROVED** [review]

**Pull request merged**